tests/modules/trecmod.nim

```diff
@@ -2,7 +2,7 @@ discard """
   errormsg: "recursive module dependency detected"
   file: "mrecmod.nim"
   line: 1
-  disabled: true
+    disabled: true
```

Comment:
not sure about this one.

Reply:
it should simply error out... I don't get why it's disabled.

Reply:
Created at 2008. 
Filled with something around 2016.
It got disabled the very next day, 
in a commit with message  `recursive modules are only detected to improve error messages`
running it gives:
```Failure: reMsgsDiffer
Expected:
recursive module dependency detected

Gotten:

```

Reply:
Let's delete it.